### ISSUE TITLE
about pop twice after selecting cause some problems

### ISSUE THREAD

**Issue by mnba3223** [hcbpassos/drag_select_grid_view#30]
HI, I am new to flutter and have some question about the pop twice after selecting.
I know when select the grid with longpress will in drag_select mode, but there's not have a onTap to switch on it.
To solve it  I use another GestureDetector with onTap to separated those two function and it work fine on select grid.

onTap like this
 GestureDetector(
              onTap: () {
                widget.selectedImages(widget.url);
                debugPrint("     ${widget.isSelected}");
                debugPrint("     ${scaleAnimation.value}");
              },
            ),

But because onTap don't need to pop twice,  when I use cancel button, I can't know which is onTap, which is longpress.

How do I know the drag_select mode is trigger or not ? 
Beacause If I knew, I can pop When I want. 

If you read this, I will appreciate you. Thank you for your reading.

**Comment by vontdeux** [comment]
did you manage to found a workaround for this?

**Comment by mnba3223** [comment]
> did you manage to found a workaround for this?

no, I keep onTap select function, not use this package, sorry for that I don't have time to fix this problem.
